src/ui/components/VPNExpandableAppList.qml

```diff
@@ -0,0 +1,143 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+import QtQuick 2.5
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtGraphicalEffects 1.15
+import Mozilla.VPN 1.0
+
+VPNClickableRow {
+    id: appRow
+
+    property var listVisible : true
+    property var listModel: undefined
+    property var header: ""
+    property var description: ""
+    property var actionText: ""
+    property var onAction: ()=>{}
+    property var isEnabled: true
+
+
+    state: (listVisible && isEnabled) ? "list-visible" : "list-hidden"
+    opacity: isEnabled ? 1 : 0.5
+
+    Keys.onReleased: if (event.key === Qt.Key_Space) handleKeyClick()
+    handleMouseClick: function() { listVisible = !listVisible; }
+    handleKeyClick: function() { listVisible = !listVisible; }
+    clip: true
+    canGrowVertical: true
+    accessibleName: name
+    states: [
+        State {
+            name: "list-hidden"
+            PropertyChanges {
+                target: toggleArrow
+                rotation: -90
+
+            }
+            PropertyChanges {
+                target: appRow
+                height: appRowHeader.height
+            }
+
+        },
+        State {
+            name: "list-visible"
+
+            PropertyChanges {
+                target: toggleArrow
+                rotation: 0
+
+            }
+            PropertyChanges {
+                target: appRow
+                height: appRowHeader.height +applist.contentHeight + applist.anchors.topMargin
+            }
+
+        }
+    ]
+
+
+    RowLayout {
+        id: appRowHeader
+        anchors.top: parent.top
+        anchors.left: parent.left
+        width: appRow.width
+        spacing: 0
+        VPNServerListToggle {
```

Comment:
Getting a `serverCountry is not defined` error here from the `serverCountry` reference on line 41 of VPNServerListToggle.qml